### PR TITLE
Add cqm models to frontend

### PR DESCRIPTION
**Pull request by lukoktonos**
This PR includes adding package.json and yarn.lock to Bonnie so we can load in npm modules.  It also includes adding and browserifying mongoose as well as adding `cqm-models` via that new mechanism, as well as a couple of tests to ensure it is working correctly.

As a side effect of loading cqm-models, cql-execution is now accessible via `cqm.models.CQL`, which is cool.  We can eventually use that instead of cql4browsers loaded in from cql_qdm_patientapi.

`yarn install` will automatically run when starting the server to ensure things are up-to-date.

`package.json` contributor list was generated based on github contribution list.

Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1912
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced. N/A
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Test fixtures updated and documented as necessary ( see [internal wiki](https://gitlab.mitre.org/bonnie/internal-documentation/wikis/testing#test-fixtures) ) N/A
- [x] Code coverage has not gone down and all code touched or added is covered. 
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here: 
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here: 
- [x] Automated regression test(s) pass N/A does not affect calculation

**Reviewer 1:**

Name: @zlister 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name: @hackrm  
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    {
+      "name": "Kristian Mulcahy",
+      "email": "kmulcahy@mitre.org"
+    },
+    {
+      "name": "Cole Springate-Combs",
+      "email": "cspringatecombs@mitre.org"
+    },
+    {
+      "name": "Cole Springate-Combs",
```
Comment (zlister) [inline package.json]: Looks like Cole is listed twice on here, not sure if this was copied from somewhere but it may also be duplicated there too

Comment (lukoktonos) [reply package.json]: Good catch.  It wasn't copied from anywhere, I typed them in from the list here https://github.com/projecttacoma/bonnie/graphs/contributors

**Review by zlister — DISMISSED** [review]

File: package.json [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    {
+      "name": "Andre Quina",
+      "email": "aquina@mitre.org"
+    },
+    {
+      "name": "Lizzie Charbonneau",
+      "email": "lizzie@mitre.org"
+    },
+    {
+      "name": "James Bradley",
```
Comment (hackrm) [inline package.json:25]: It also looks like James was added twice (lines 25 & 32).

Comment (lukoktonos) [reply package.json:25]: Good catch! Fixed by removing the one below.

**Review by hackrm — APPROVED** [review]

**Review by zlister — APPROVED** [review]

**Pull request merged**